distributed/tests/test_client.py

```diff
@@ -6891,8 +6891,8 @@ async def f(stacklevel, mode=None):
 
 
 @pytest.mark.skipif(
```

Comment:
This, and `test_client_connectionpool_semaphore_loop`, are always being skipped now. @fjetter are these tests actually useful? I'd prefer to drop if not

Reply:
no, they can go away